### PR TITLE
[13_0_X]Add WFs for 2023D

### PR DESCRIPTION
**Pull request by sunilUIET**
Backport of https://github.com/cms-sw/cmssw/pull/42413

### CODE REVIEW

**Comment by sunilUIET** [comment]
test parameters:
relvals_opt= -w standard
workflows = 141.039,141.041,141.041,141.043,141.044,141.045,141.046,141.047,141.048,141.049

**Comment by sunilUIET** [comment]
please test

**Comment by sunilUIET** [comment]
+pdmv

**Comment by sunilUIET** [comment]
test parameters:
relvals_opt= -w standard
workflows = 141.039,141.041,141.041,141.043,141.044,141.045,141.046,141.047,141.048,141.049
pull_request=https://github.com/cms-sw/cmssw/pull/42372

**Comment by sunilUIET** [comment]
please test

File: Configuration/PyReleaseValidation/python/relval_steps.py [hunk 1]
```diff
@@ -597,6 +597,21 @@
 steps['RunMuonEG2023C']={'INPUT':InputInfo(dataSet='/MuonEG/Run2023C-v1/RAW',label='2023C',events=100000,location='STD', ls=Run2023C)}
 steps['RunParkingDoubleMuonLowMass2023C']={'INPUT':InputInfo(dataSet='/ParkingDoubleMuonLowMass0/Run2023C-v1/RAW',label='2023C',events=100000,location='STD', ls=Run2023C)}
 
+Run2023C_LS500={368489: [[46,546]]}
+steps['TestEnablesEcalHcal2023C']={'INPUT':InputInfo(dataSet='/TestEnablesEcalHcal/Run2023C-Express-v4/RAW',label='2023C',events=100000,location='STD', ls=Run2023C_LS500)}
+
```
Comment (perrotta) [inline Configuration/PyReleaseValidation/python/relval_steps.py:602]: This is from #41607, not needed in 13_0_X without the rest of that PR

```suggestion
```

**Comment by sunilUIET** [comment]
+pdmv

**Comment by perrotta** [comment]
+1
- This was already signed bu @cms-sw/upgrade-l2 , see https://github.com/cms-sw/cmssw/pull/42416#issuecomment-1658607804
- Now the update is identical to the ones already merged in master and in 13_2_X: I consider that previous signature still valid

**Comment by perrotta** [comment]
merge

**Pull request merged**